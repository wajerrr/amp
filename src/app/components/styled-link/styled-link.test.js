import React from 'react';
import renderer from 'react-test-renderer';
import StyledLink from './styled-link';

describe('StyledLink', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<StyledLink href="economist.com">click me</StyledLink>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
