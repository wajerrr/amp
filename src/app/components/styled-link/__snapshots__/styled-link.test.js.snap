// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledLink should match snapshot 1`] = `
<a
  className="css-1h5t0lq-StyledLink e2cf1b50"
  href="economist.com"
>
  click me
</a>
`;
